src/components/Comments.jsx

```diff
@@ -33,6 +58,7 @@ export default function Comments({id, comments, setComments}) {
                         <p id="commentVotes">{comment.votes} votes</p>
                         <p id="commentBody">{comment.body}</p>
                         <date>{postedAt}</date>
```

Comment:
This date element is throwing an error in React as it isn't a valid HTML element. If you check your console, you should see it. Can you find an alternative element for date information that exists?